cmd/genpartial/main.go

```diff
@@ -50,9 +50,8 @@ func main() {
 	}
 
 	pkgs := Must(packages.Load(&packages.Config{
-		Dir:        cwd,
-		Mode:       mode,
-		BuildFlags: []string{"-tags=generate"},
+		Dir:  cwd,
+		Mode: mode,
```

Comment:
Is this change intentional?

Reply:
It is intentional as Redpanda.Values have now console.PartialValues. As the PartialValues struct is in the file that have generated build tag, the load package needs that.

https://github.com/redpanda-data/helm-charts/blob/0ea509830613d936b213dae497e72356fa858485/charts/console/values_partial.gen.go#L1

Reply:
Should we also remove the build directive from the generation as nothing will be using that now?